### PR TITLE
DEVADV-191 validating and reviewing the start-using-sdk page for .NET

### PR DESCRIPTION
**Pull request by mgroves**
This is my attempt at validating and tweaking the start-using-sdk page for .NET

### CODE REVIEW

File: modules/hello-world/pages/start-using-sdk.adoc [hunk 1]
```diff
@@ -3,34 +3,36 @@
...
+The Couchbase .NET client allows applications to connect to Couchbase Server using any Common Language Runtime (CLR) language, including C#, F#, and VB.NET. 
 The SDK is written in C#, and some of its idiomatic patterns reflect that choice.
-The .NET SDK 3.0 is a complete rewrite of the API, reducing the number of overloads to present a simplified surface area, and adding support for future Couchbase Server features like Collections and Scopes (available in Couchbase Server 6.5 as a xref:concept-docs:collections.adoc[developer preview]).
-The API is now fully async.
-
 
 == Installing the SDK
 
-The Couchbase .NET SDK targets https://docs.microsoft.com/en-us/dotnet/standard/net-standard[.NET Standard 2.0], which in turn supports .NET Framework 4.6.2+ and .NET Core (eg netcoreapp) 2.1. The library is distributed in a number of ways:
+The Couchbase .NET SDK targets https://docs.microsoft.com/en-us/dotnet/standard/net-standard[.NET Standard 2.0], which in turn supports .NET Framework 4.6.2+ and .NET Core (e.g. netcoreapp) 2.1.
```
Comment (RichardSmedley) [inline modules/hello-world/pages/start-using-sdk.adoc:14]: Has this changed to Core 3.0/3.1?

**Review by RichardSmedley — COMMENTED** [review]
LGTM - but while we're updating the page this would be a good opportunity to clarify if the .NET Core version supported needs updating.   
Otherwise :+1: 
Thanks for the contribution @mgroves

**Review by RichardSmedley — APPROVED** [review]
(but not going to hold up the merge for that :) )

File: modules/hello-world/pages/start-using-sdk.adoc (hunk 1 above)
Comment (jeffrymorris) [reply modules/hello-world/pages/start-using-sdk.adoc:14]: We target .netstandard2.0, .netstandard2.1, .netcoreapp2.1 and .netcoreapp3.0.  Note that .netcoreapp2.1 and .netcoreapp3.0 are so we can use libraries not yet part of .netstandard; these are for provided if you would wish to take advantage of the libraries as they are primarily performance optimizations. 

- .netstandard2.0 targets .NET Core 2.0 and up and .NET Framework 4.6.1 and greater (plus mono 5.4 and greater)

- .netstandard2.1 targets .NET Core 3.0 and greater (plus Mono 6.4, etc)

This page should help: https://dotnet.microsoft.com/platform/dotnet-standard

Comment (RichardSmedley) [reply modules/hello-world/pages/start-using-sdk.adoc:14]: Thanks @jeffrymorris - will try and get this into a separate PR today, once I've digested it. ;)

File: modules/hello-world/pages/start-using-sdk.adoc [hunk 2]
```diff
@@ -3,34 +3,36 @@
...
 == Installing the SDK
 
-The Couchbase .NET SDK targets https://docs.microsoft.com/en-us/dotnet/standard/net-standard[.NET Standard 2.0], which in turn supports .NET Framework 4.6.2+ and .NET Core (eg netcoreapp) 2.1. The library is distributed in a number of ways:
+The Couchbase .NET SDK targets https://docs.microsoft.com/en-us/dotnet/standard/net-standard[.NET Standard 2.0], which in turn supports .NET Framework 4.6.2+ and .NET Core (e.g. netcoreapp) 2.1. 
+The library is distributed in a number of ways:
 
 |===
-|Nuget |Nuget package host at https://www.nuget.org/packages/CouchbaseNetClient/[nuget.org] (_recommended_)
+|NuGet |NuGet package host at https://www.nuget.org/packages/CouchbaseNetClient/[nuget.org] (_recommended_)
 |Zip |Zip file with can be downloaded https://packages.couchbase.com/clients/net/3.0/Couchbase-Net-Client.3.0.0-beta2.zip[here]
```
Comment (blbryant) [inline modules/hello-world/pages/start-using-sdk.adoc:19]: DA questions the use of the .zip file download (perhaps it should be removed: DOC-6844).  If we do not remove it, can we at least move it to the bottom of the list?

**Pull request merged**